src/containers/Analytics/DataTable.tsx

```diff
@@ -35,8 +35,12 @@ export const DataTable = ({ title, colums, rows }: TableProps) => {
                                 {item?.map((value, valueIndex) => (
```

Comment:
Can you add logic here to skip the item in the list? Or remove the item from the list prior to this mapping

Reply:
I've skipped item with index 6, but I don't like this solution. I can dig dipper, or can leave it like this.

Reply:
Found where it comes from. So I removed it from fetching
```
const colRows = Object.fromEntries(
                        Object.entries(analyticsData?.tokenAnalyticsData).map(([key, value]) => {
                            const lockedAmountInUsd = multiplyWad(
                                value?.lockedAmount?.toString(),
                                value?.currentPrice?.toString()
                            )
                            totalLockedValue = totalLockedValue.add(lockedAmountInUsd)

                            return [
                                key,
                                [
                                    key,
                                    <AddressLink address={geb.tokenList[key].address} chainId={chainId || 420} />,
                                    <AddressLink address={value?.delayedOracle} chainId={chainId || 420} />,
                                    formatDataNumber(value?.currentPrice?.toString() || '0', 18, 2, true),
                                    formatDataNumber(value?.nextPrice?.toString() || '0', 18, 2, true),
                                    transformToAnnualRate(
                                        multiplyRates(
                                            value?.stabilityFee?.toString(),
                                            analyticsData.redemptionRate?.toString()
                                        ) || '0',
                                        27
                                    ),
                                    formatDataNumber(value?.debtAmount?.toString() || '0', 18, 2, true, true),
                                    transformToWadPercentage(
                                        value?.debtAmount?.toString(),
                                        value?.debtCeiling?.toString()
                                    ),
                                    formatDataNumber(value?.lockedAmount?.toString() || '0', 18, 2, false, true) +
                                        ' ' +
                                        key,
                                    formatDataNumber(
                                        multiplyWad(value?.lockedAmount?.toString(), value?.currentPrice?.toString()) ||
                                            '0',
                                        18,
                                        2,
                                        true,
                                        true
                                    ),
                                    transformToWadPercentage(
                                        multiplyWad(
                                            value?.debtAmount?.toString(),
                                            analyticsData?.redemptionPrice?.toString()
                                        ),
                                        multiplyWad(value?.lockedAmount?.toString(), value?.currentPrice?.toString())
                                    ),
                                ],
                            ]
                        })
                    )
```